tests/e2e-tests/fixtures/fixture-data.js

```diff
@@ -0,0 +1,192 @@
+const Coupons = () => [
```

Comment:
Might be worth documenting here that the shape of all this data is dictated by the relevant APIs.

Not sure if its worthwhile making them all functions – most are not dynamic. (Not a blocker just a thought)

Reply:
However we don't know when or how this data might get dynamic, some coupons might need certain products IDs or Categories, shipping might need a class that is created in Products, Products will need reviews, there's always overlap, so I wanted to keep the interface the same, so all fixtures are a function, this should make using them more predictable.